src/templates/haml/home5/index.html.haml

```diff
@@ -104,11 +106,26 @@
     .home-content #open-account .input-button:hover {
       background: #2a3052;
       border: solid 2px #fff;
-      margin-top: -2px !important;
+      margin-top: 27px !important;
       margin-bottom: -2px !important;
     }
+    .home-content #open-account .open-account-sub-text{
+      font-size:0.7em;
+      display: inline-block;
+      margin-top: 10px;
+      }
+      .fixed-h-input{
+        height: 40px;
+        overflow: hidden;
+      }
+      .second-label{
+        margin:4px 0 0 0;
+      }
+      .home-content #banner{
+
+      }
 .home-content.paragraph-style
-  #banner.section-margin{:style=>'background-position-y: 68%;background-position-x: -272px;'}
+  #banner.section-margin{:style=>'background: url(../images/pages/home5/banner-background_1.png);background-repeat: no-repeat;background-position: 50% 0%;'}
```

Comment:
I don't think url will work. We need to use url_for I think. Can you check if we have used url instead of url_for?